# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/transfig/use-stdarg-h-gcc33-fix.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- transfig.3.2.4/fig2dev/fig2dev.h	2002-12-18 23:45:28.000000000 +0100
+++ transfig.3.2.4-jsaw/fig2dev/fig2dev.h	2004-03-30 12:28:56.000000000 +0200
@@ -22,7 +22,7 @@
 #include <sys/file.h>
 #include <signal.h>
 #include <string.h>
-#include <varargs.h>
+#include <stdarg.h>
 #include <pwd.h>
 #include <errno.h>
 #include <time.h>
